strongbox-web-core/src/test/java/org/carlspring/strongbox/rest/RestAssuredExampleTest.java

```diff
@@ -0,0 +1,46 @@
+package org.carlspring.strongbox.rest;
+
+import org.carlspring.strongbox.rest.context.RestletTestContext;
+
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import static io.restassured.RestAssured.get;
+
+/**
+ * @author Alex Oreshkevich
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@RestletTestContext
+//@WebAppConfiguration
+@Ignore
+public class RestAssuredExampleTest
+{
+
+    @BeforeClass
+    public static void setupRA() {
+        RestAssured.baseURI = "http://localhost";
+        RestAssured.port = 8080;
```

Comment:
If we pick RestAssured, the port here will have to be configurable, like it's done here [here](https://github.com/strongbox/strongbox/blob/master/strongbox-client/src/main/java/org/carlspring/strongbox/client/ArtifactClient.java#L44).